src/azul/project/hca/transformers.py

```diff
@@ -409,8 +473,11 @@ def transform(self,
             file.ancestors(visitor)
         project = self._get_project(bundle)
 
-        contents = dict(specimens=[_specimen_dict(s, visitor.visited_entities) for s in visitor.specimens.values()],
+        contents = dict(specimens=[_specimen_dict(s) for s in visitor.specimens.values()],
```

Comment:
Both project and file documents should contain `content.samples`. That way we can track what sample types a project or file is associated with to.